spring-core/src/test/java/org/springframework/core/serializer/SerializerTests.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2002-2022 the original author or authors.
```

Comment:
```suggestion
 * Copyright 2002-2023 the original author or authors.
```

Please ensure all created/updated files have the correct copyright year.

Reply:
Added the changed.